data/sql_setup/prepare_schedule_a.sql

```diff
@@ -96,7 +96,14 @@ begin
         -- visit here:
         -- https://www.postgresql.org/docs/current/static/plpgsql-statements.html#PLPGSQL-STATEMENTS-DIAGNOSTICS
         if FOUND then
-            two_year_transaction_period = get_transaction_year(new.contb_receipt_dt, view_row.rpt_yr);
+            -- Requires that the hstore extension is installed.
+            if hstore(new) ? 'contb_receipt_dt' then
+                contb_receipt_dt = new.contb_receipt_dt;
+            else
+                contb_receipt_dt = cast(null as timestamp);
```

Comment:
@LindsayYoung:  Instead of just ignoring the value here (and in the other areas where this fix has been made), should we leverage the `rpt_yr` value in the `disclosure.f_item_receipt_or_exp` table instead and use that before falling back to the row in the view?